Extension/src/common.ts

```diff
@@ -1292,3 +1288,11 @@ export async function checkCuda(): Promise<void> {
     const langs: string[] = await vscode.languages.getLanguages();
     supportCuda = langs.findIndex((s) => s === "cuda-cpp") !== -1;
 }
+
+// Sequentially Resolve Promises.
+export async function sequentialResolve(items: any[], promiseBuilder: (item: any) => Promise<any>): Promise<void> {
```

Comment:
To improve type safety with this method, can you change the signature to:
```
export async function sequentialResolve<T>(items: T[], promiseBuilder: (item: T) => Promise<void>): Promise<void> {
```

(basically remove the `any` types)